src/main/java/net/fabricmc/loader/game/MinecraftGameProvider.java

```diff
@@ -158,6 +159,31 @@ public void acceptArguments(String... argStrs) {
 		FabricLauncherBase.processArgumentMap(arguments, envType);
 	}
 
+	@Override
+	public String[] getLaunchArguments(boolean sanitize) {
+		if (arguments != null) {
+			List<String> list = new ArrayList<>(Arrays.asList(arguments.toArray()));
+			if (sanitize) {
+				int remove = 0;
+				Iterator<String> iterator = list.iterator();
+				while (iterator.hasNext()) {
+					String next = iterator.next();
+					if ("--accessToken".equals(next)) {
+						remove = 2;
+					}
+
+					if (remove > 0) {
+						iterator.remove();
+						remove--;
+					}
+				}
+			}
+			return list.toArray(new String[0]);
+		} else {
```

Comment:
you can drop the `else` here and just return the empty string array at the bottom of the method.

Reply:
Done!